swift/tools/swiftbuild.py

```diff
@@ -147,4 +138,17 @@ def build_package(test, run, package, swiftc_options):
     if args.parseable:
         swiftc_options.append("-parseable-output")
 
+    # If no package list provided, use all directories in the
+    # current working directory.
+
+    if len(args.packages) == 0:
+        cwd_path = pathlib.Path().absolute()
```

Comment:
This will be relative to the cwd, which could be several places. I've been using `Path(__file__).absolute().parent.parent` for python scripts in a `tools` directory, so that the root is consistent regardless of where it was invoked from.